docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: https://github.com/Luzifer/ots.git#v1.8.0
```

Comment:
Any idea how we can keep this to the latest tag? Is there some `#latest` like Go has its `go install ...@latest`? I know myself and I'm 100% sure this will be outdated next release…

Reply:
Just use `#master`

Reply:
Hmm I think `#master` isn't the best variant: It will build the development version, not the latest release… Sadly looking at the [docker build reference](https://docs.docker.com/engine/reference/commandline/build/#git-repositories) there doesn't seem something like `#latest`… :/

Reply:
I was also thinking about preparing a pipeline triggered by a new tag being pushed that'll automagically update the `docker-compose` if the reference is different than incoming tag - `yq` might be used to make necessary changes. After pushing such change, the tag could be updated.

Reply:
Hmm yeah, in order not to add another commit on top of the tag I could add a script hooking into my release command updating the `docker-compose.yml` with the release commit… Had hoped to get around that… :D